### DISCUSSION TITLE
No way to get data from basen bms 200A 16s

### DISCUSSION THREAD

**Discussion by karibe66** [syssi/esphome-basen-bms#27] [General]
Hallo and sorry for my request. I'm trying to connect via esp32 with Bms Basen 200A 16s. I can connect for few seconds but than it lose connection. I paste a log from my laptop. Can someone please help me?
Thanks
![1701449826345](https://github.com/syssi/esphome-basen-bms/assets/22154260/2ab025aa-f591-4fcc-af3e-f0833acfca98)

**Comment by syssi** [comment]
It looks like there is something different with your BMS. Either the BLE module has changed or your battery is equipped with a different BMS. Could you increase the log level (see https://github.com/syssi/esphome-basen-bms?tab=readme-ov-file#debugging) an try again? I would like to see some more details about the BLE services & chars. 

Could you provide a link to the Android app which supports your BMS?

**Comment by karibe66** [comment]
Hi Sebastian, and thank you very much.
You are right, even if the battery pack is from Basen the Bms is a
Tianpower TP-LT55TB.
I change configuration yaml using always your configuration for a Tianpower
It works very well!!
Thank you so much for the great job you are doing

Happy new year
Mario

**Reply by syssi** [reply]
Perfect! Let's close this issue.

**Reply by syssi** [reply]
@karibe66 I would like to add your battery bank / BMS model to the list of supported devices. Could you provide some details about your model?
